src/js/components/Button/doc.js

```diff
@@ -102,6 +113,10 @@ with plain Buttons.`,
               end of the anchor.`,
       )
       .defaultValue(false),
+    simple: PropTypes.bool.description(
```

Comment:
we're thinking of having `emphasis` prop with either strong or weak values. strong will be translated to `primary` while `weak` will be translated to `simple`. 
we can replace the existing `primary` examples to show the usage of the new prop while still supporting `primary` for backward compatibility.